bin/mustache_cli.ml

```diff
@@ -11,6 +11,15 @@ let load_file f =
   close_in ic;
   (Bytes.to_string s)
 
+let locate_template search_path filename =
+  if not (Filename.is_relative filename) then begin
+    if Sys.file_exists filename then Some filename
+    else None
+  end else
```

Comment:
Could you reverse this to get rid of the not? You'll also be able to rewrite it more naturally:

```
if .. then
 ..
else if Sys.file_exists .. then
 ..
else
 None
```

Reply:
Done. (I had a slight preference for handling the absolute-path case first, as it is simpler, but indeed having two nested ifs less pleasant.)